modules/custom/d_content/config/update/d_content.8118.yml

```diff
@@ -0,0 +1,12 @@
+core.entity_form_display.node.content_page.default:
+  add:
+    content:
+      field_header_layout:
+        region: content
+        type: options_select
+field.field.node.content_page.field_header_layout:
+  change:
+    expected:
+      required: false
+    new:
+      required: true
```

Comment:
tam już jest plik z update, tylko w ferworze naprawiania zostało wyłączone wciąganie go.
Pa:
![image](https://user-images.githubusercontent.com/37533838/88022067-e2123680-cb2e-11ea-9d61-d71ab5d82dd1.png)

Wystarczyłoby chyba to przenieść do arrayki poniżej:
![image](https://user-images.githubusercontent.com/37533838/88022139-053ce600-cb2f-11ea-9c66-ac2b5c03632a.png)

Reply:
ok, to akurat zmienilem u siebie w PR.. duzo tam w updatach grzebalem.. moze byc nieaktualne jak moj PR pierwszy wejdzie :)